packages/flutter/lib/src/cupertino/activity_indicator.dart

```diff
@@ -18,6 +18,15 @@ const Color _kActiveTickColor = CupertinoDynamicColor.withBrightness(
   darkColor: Color(0xFFEBEBF5),
 );
 
+/// Define the style of [CupertinoActivityIndicator].
+enum CupertinoActivityIndicatorStyle {
```

Comment:
I recommend that we remove this enum and `style` flags and only support iOS 14.  If we wait a few weeks to merge this PR so that it misses the next stable release, then the following stable release should happen pretty close to September and the likely release of iOS 14.

Flutter's usual policy is to support only the latest version of a design style.  For example, see https://github.com/flutter/flutter/pull/36087 where switches were updated to match iOS 13's style several months before the release of iOS 13.

Otherwise, if we really want to support both versions right now, we need to have a plan for deprecation. That probably involves creating another issue and adding TODOs to the code saying that it needs to be deprecated. Then someone needs to merge a PR to deprecate this, and then later another PR to remove the iOS 13 code altogether.

Reply:
I think developers need time to adapt the app for iOS14 and user also need time to update to iOS14. It will be better to support the new style before official release of iOS14 to give developers enough time to adapt their UI.

I've added a TODO for changing default style to iOS14. (Maybe I will do that.)

Reply:
Sounds good, if you want to do it this way then I'm on board with that.  Thanks for adding the TODO.  I think the deprecation process should look like this:

Roughly 1 month before the release of iOS 14, we should merge another PR that adds the `@deprecated` flag to the `style` parameter.  This PR should also change the default value to `CupertinoActivityIndicatorStyle.iOS14`.  This will have the effect of automatically updating users to iOS 14 style if they haven't specifically set their style.  The PR should be merged 1 month before iOS 14's release so that the PR is released to stable around the same time.

After iOS 14 is released and everything is working well with the new indicator style, we should merge another PR that removes the `style` parameter and `CupertinoActivityIndicatorStyle` altogether.  This will force everyone to the iOS 14 style.

Reply:
We can either keep https://github.com/flutter/flutter/issues/60047 open until the final step is complete, or we can create 2 new issues to track the two points above.

Reply:
Sure, we can keep the old issue and update the progress.